application.go

```diff
@@ -49,6 +49,15 @@ type OAuthResponse struct {
 	StormpathAccessTokenHref string `json:"stormpath_access_token_href"`
 }
 
+type AccessToken struct {
+	resource
+	Account      resource               `json:"account,omitempty"`
```

Comment:
This should be actual pointers to the actual type Account Tenant and Application